experimental/GModule/src/Cohomology.jl

```diff
@@ -444,7 +444,10 @@ function Oscar.tensor_product(C::GModule{<:Any, FinGenAbGroup}...; task::Symbol
   end
 end
 
-function Oscar.tensor_product(C::GModule{S, <:AbstractAlgebra.FPModule{<:Any}}...; task::Symbol = :map) where S <: Oscar.GAPGroup
+function Oscar.tensor_product(C::U, Cs::U...; task::Symbol = :map) where {S <: Oscar.GAPGroup, U <: GModule{S, <:AbstractAlgebra.FPModule{<:Any}}}
+  return Oscar.tensor_product([C, Cs...])
```

Comment:
```suggestion
  return Oscar.tensor_product([C, Cs...]; task)
```
The kwargs should be forwarded